guides/common/modules/proc_installing-the-iop-advisor-engine-using-import-and-export.adoc

```diff
@@ -0,0 +1,29 @@
+[id="installing-the-iop-advisor-engine-using-import-and-export"]
+= Installing the IoP Advisor Engine using import and export
+
+Use this procedure to transfer the IoP Advisor Engine container image from a connected system to a disconnected {ProjectServer}. 
+
+.Procedure
+To transfer and install the container image from a connected system:
+
+. On the connected system, export the container image:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# skopeo copy oci-archive:/tmp/iop-advisor-engine.tar \
```

Comment:
I don't think Skopeo is installed by default. Installing it might be a prerequisite.